src/server/external-systems/van.ts

```diff
@@ -137,16 +335,165 @@ const VAN: IExternalSystem = {
       .where({ id: syncId });
   },
 
-  async syncQuestionResponse(payload: Record<string, any>): Promise<void> {
-    console.log("***********************");
-    console.log(payload);
-    console.log("***********************");
+  async syncQuestionResponse(
+    payload: Record<string, any>,
+    helpers: JobHelpers
+  ): Promise<void> {
+    const { campaignContactId, externalSystemId } = payload;
+
+    const campaignContact = await r
+      .knex("campaign_contact")
+      .where({ id: campaignContactId })
+      .first();
+
+    // Get all actions that need to be synced
+    // Ignores any that is not mapped
```

Comment:
I envision 3 scenarios here:

1. Normal conversation, a question response is added, and no changes in the next 10 mins to this conversation. Single question response is synced
2. Super active conversation, multiple question responses are added, all actions are synced together 10 mins after last update. This prevents multiple jobs being setup, since VAN support batch inserts for multiple responses, this works well. 
3. Change by a sweeper or admin through message review, question responses were marked wrong, new actions are synced together 10 mins after the last update. Batch sync because it's supported by VAN. 

ActionKit or other external systems might not support batch sync, and since VAN supports it, I wanted to take advantage of it, and not sync one action per job which on an active campaign can potentially cause lots of performance issues to work through.